include/swift/AST/Module.h

```diff
@@ -966,13 +967,10 @@ class SourceFile final : public FileUnit {
                                              SourceLoc diagLoc = {});
   /// @}
 
-  ReferencedNameTracker *getReferencedNameTracker() const {
-    return ReferencedNames;
-  }
-  void setReferencedNameTracker(ReferencedNameTracker *Tracker) {
-    assert(!ReferencedNames && "This file already has a name tracker.");
-    ReferencedNames = Tracker;
+  ReferencedNameTracker *getReferencedNameTracker() {
+    return ReferencedNames ? ReferencedNames.getPointer() : nullptr;
   }
+  void createReferencedNameTracker(StringRef);
```

Comment:
Missing argument name again!

Reply:
Fixed! In this case the argument name clarifies things. In hypothetical other cases where it would be redundant, would the common practice here be to include it anyway?

Reply:
Yes, I think so. I think that's just always the convention, at least for top-level arguments.